From d4bcb0003963156e22ea52bfecc0e6335b504bec Mon Sep 17 00:00:00 2001
From: bivsk <bivsk@tutanota.com>
Date: Fri, 16 Jan 2026 23:26:24 -0500
Subject: [PATCH] do not vendor randomx

---
 build.rs | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/build.rs b/build.rs
index 7cb42dc..84b971c 100644
--- a/build.rs
+++ b/build.rs
@@ -26,10 +26,10 @@ use cmake::Config;
 
 #[allow(clippy::too_many_lines)]
 fn main() {
-    let randomx_path = Config::new(env::var("RANDOMX_DIR").unwrap_or_else(|_| "RandomX".to_string()))
-        .define("DARCH", "native")
-        .build();
-    println!("cargo:rustc-link-search=native={}/lib", randomx_path.display());
+    // let randomx_path = Config::new(env::var("RANDOMX_DIR").unwrap_or_else(|_| "RandomX".to_string()))
+    //     .define("DARCH", "native")
+    //     .build();
+    println!("cargo:rustc-link-search=native={}/lib", env::var("RANDOMX_DIR").unwrap());
     println!("cargo:rustc-link-lib=static=randomx");
     let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap_or("linux".to_string());
     let dylib_name = match target_os.as_str() {
-- 
2.52.0

